src/status_im/network/core.cljs

```diff
@@ -128,28 +128,32 @@
-                                            [::connect-failure {:network-id network-id
-                                                                :on-failure on-failure
-                                                                :reason     (i18n/label :t/network-invalid-url)}]))
-                 :failure-event-creator (fn [{:keys [response-body status-code]}]
-                                          (let [reason (if status-code
-                                                         (i18n/label :t/network-invalid-status-code {:code status-code})
-                                                         (str response-body))]
-                                            [::connect-failure {:network-id network-id
-                                                                :on-failure on-failure
-                                                                :reason     reason}]))}}
+    (if-let [upstream-url (get-in config [:UpstreamConfig :URL])]
+      {:http-post {:url                   upstream-url
+                   :data                  (types/clj->json {:jsonrpc "2.0"
+                                                            :method  "web3_clientVersion"
+                                                            :id      1})
+                   :opts                  {:headers {"Content-Type" "application/json"}}
+                   :success-event-creator (fn [{:keys [response-body]}]
+                                            (if-let [client-version (:result (http/parse-payload response-body))]
+                                              [::connect-success {:network-id     network-id
+                                                                  :on-success     on-success
+                                                                  :client-version client-version}]
+                                              [::connect-failure {:network-id network-id
+                                                                  :on-failure on-failure
+                                                                  :reason     (i18n/label :t/network-invalid-url)}]))
+                   :failure-event-creator (fn [{:keys [response-body status-code]}]
+                                            (let [reason (if status-code
+                                                           (i18n/label :t/network-invalid-status-code {:code status-code})
+                                                           (str response-body))]
+                                              [::connect-failure {:network-id network-id
+                                                                  :on-failure on-failure
+                                                                  :reason     reason}]))}}
+      (connect-success cofx {:network-id     network-id
+                             :on-success     on-success
+                             :client-version ""}))
```

Comment:
no client version ?

Reply:
I don't know what does this client version even mean, frankly.
If you run a light node yourself (LES or ULC), you can connect to all sorts of clients, different implementations and different versions, so this field makes no sense.